tests/k8st/test_base.py

```diff
@@ -182,3 +183,32 @@ def is_it_gone_yet(res_name, res_type):
 
     class StillThere(Exception):
         pass
+
+    def get_routes(self):
+        return run("docker exec kube-node-extra ip r")
+
+    def update_ds_env(self, ds, ns, k, v):
```

Comment:
Not for this PR, but it might be nice to support a list of env vars to set if we ever want to update multiple things.

Just because it takes a long time to wait for each rolling update to occur.